cpp/Contact.h

```diff
@@ -86,22 +86,35 @@ class Contact
 {
 public:
   /// Constructor
-  /// @param[in] marker The meshtags defining the contact surfaces
-  /// @param[in] surfaces Array of the values of the meshtags marking the
-  /// surfaces
+  /// @param[in] markers List of meshtags defining the contact surfaces
+  /// @param[in] surfaces Adjacency list. Links of i contains meshtag values
+  /// associated with ith meshtag in markers
+  /// @param[in] contact_pairs list of pairs (i, j) marking the ith and jth
+  /// surface in surfaces->array() as a contact pair
```

Comment:
I don't understand this datatype. Could you explain it in detail when you come in?